docs/metrics/index.md

```diff
@@ -86,6 +86,13 @@ biases, and its intended uses.
     emphasizing larger errors more significantly than smaller ones. It is mainly different from
     [MAE](mean-absolute-error.md) in that larger errors are penalized more heavily.
 
+- [Pearson Correlation Coefficient](pearson-correlation.md)
```

Comment:
Do these page titles need "Coefficient" or can that be omitted? Looking at "Cohen's Kappa" which is the closest analog in the other metrics, it seems that the more standard thing to do would be to just call these "Pearson Correlation" and "Spearman's Rank Correlation".